src/mui/form/SimpleForm.js

```diff
@@ -32,7 +32,8 @@ export class SimpleForm extends Component {
                             input &&
                             <div
                                 key={input.props.source}
-                                className={`aor-input-${input.props.source}`}
+                                className={`aor-input aor-input-${input.props
+                                    .source}`}
```

Comment:
weird new line

Reply:
Using [classnames](https://github.com/JedWatson/classnames) would resolve this and make it easier to read. Agreed ?

Reply:
Actually, I introduced a new `FormInput` component which also avoid repeating some code in `SimpleForm` and `FormTab`